webpack.config.js

```diff
@@ -1,23 +1,57 @@
+/* eslint-disable global-require */
```

Comment:
What's the need for this?

Reply:
regarding `/* eslint-disable global-require */`, I was having trouble with a `require()` throwing an eslint error, but now it is gone after removing that disable global comment.  However, I have no idea why. 🤷‍♀️